docker-compose.sandbox.yml

```
@@ -0,0 +1,7 @@
+services:
+  sandbox:
+    build:
+      context: .
+    command: ["sh", "./scripts/build-out.sh"]
+    volumes:
+      - .:/app
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)